### PR TITLE
test: convert a portion of tests from easymock to mockito

### PR DESCRIPTION
**Pull request by dmitry-fa**
Toward  #270 

For the convenience of reviewers this PR comprises two commits, the second commit allows to see the converted tests as if there were changed, not added.

### CODE REVIEW

File: google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java [hunk 1]
```diff
@@ -1637,4 +1637,459 @@ public void testWriterFailure() {
...
+        new StorageRpc.RewriteResponse(rpcRequest, null, 42L, false, "token", 21L);
+    doReturn(rpcResponse, rpcResponse)
+        .doThrow(UNEXPECTED_CALL_EXCEPTION)
+        .when(storageRpcMock)
+        .openRewrite(rpcRequest);
+    initializeService();
+    CopyWriter writer = storage.copy(request);
+    assertEquals(42L, writer.getBlobSize());
+    assertEquals(21L, writer.getTotalBytesCopied());
+    assertTrue(!writer.isDone());
```
Comment (elharo) [inline google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java:1945]: assertFalse(writer.isDone())

File: google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java [hunk 2]
```diff
@@ -1637,4 +1637,459 @@ public void testWriterFailure() {
...
+    request =
+        Storage.CopyRequest.newBuilder()
+            .setSource(BLOB_INFO2.getBlobId())
+            .setSourceOptions(Storage.BlobSourceOption.decryptionKey(BASE64_KEY))
+            .setTarget(BLOB_INFO1, Storage.BlobTargetOption.encryptionKey(KEY))
+            .build();
+    writer = storage.copy(request);
+    assertEquals(42L, writer.getBlobSize());
+    assertEquals(21L, writer.getTotalBytesCopied());
+    assertTrue(!writer.isDone());
```
Comment (elharo) [inline google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java:1955]: assertFalse

File: google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java [hunk 3]
```diff
@@ -1637,4 +1637,459 @@ public void testWriterFailure() {
...
+        new StorageRpc.RewriteResponse(rpcRequest, null, 42L, false, "token", 21L);
+    doReturn(rpcResponse)
+        .doThrow(UNEXPECTED_CALL_EXCEPTION)
+        .when(storageRpcMock)
+        .openRewrite(rpcRequest);
+    initializeService();
+    CopyWriter writer = storage.copy(request);
+    assertEquals(42L, writer.getBlobSize());
+    assertEquals(21L, writer.getTotalBytesCopied());
+    assertTrue(!writer.isDone());
```
Comment (elharo) [inline google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java:2023]: assertFalse

File: google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java [hunk 4]
```diff
@@ -1637,4 +1637,459 @@ public void testWriterFailure() {
...
```
Comment (elharo) [inline google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java:2093]: assertFalse

Comment (dmitry-fa) [reply google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java]: @elharo assertTrue changed to assertFalse when needed, please take a look.

Comment (dmitry-fa) [reply google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java]: But now it causes linkage-monitor failing:

```
* Connection #0 to host storage.googleapis.com left intact
+ java -jar linkage-monitor-latest-all-deps.jar com.google.cloud:libraries-bom
Error: Exception in thread "main" java.lang.IllegalArgumentException: The option 'send-analytics' contains an illegal character : '-'
	at org.apache.commons.cli.OptionValidator.validateOption(OptionValidator.java:71)
	at org.apache.commons.cli.Option.<init>(Option.java:146)
	at org.apache.commons.cli.Options.addOption(Options.java:150)
	at org.apache.commons.cli.Options.addOption(Options.java:131)
	at com.google.cloud.tools.dependencies.linkagemonitor.LinkageMonitor.parseCommandLine(LinkageMonitor.java:124)
	at com.google.cloud.tools.dependencies.linkagemonitor.LinkageMonitor.main(LinkageMonitor.java:96)
Error: Process completed with exit code 1.
```

Comment (dmitry-fa) [reply google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java]: @elharo could you take a look, please

**Pull request closed without merging**